### ISSUE TITLE
check_ping times out when host is down

### ISSUE THREAD

**Issue by bardahlm** [nagios-plugins/nagios-plugins#323]
#139 changed check_ping from using -w timeout to -W timeout. -W, however,  sets the timeout for each packet, not the timeout for the entire ping process.

As of now, all checks on hosts that are down report
`CRITICAL - Plugin timed out`
instead of
`PING CRITICAL - Packet loss = 100%|rta=500.000000ms;300.000000;500.000000;0.000000 pl=100%;20;25;0`

**Comment by prakashveni1** [comment]
Hi Team,

As i configured all hosts but i could not able to ping my client hist in nagios server, shows CRITICAL - Plugin timed out. could any one assist plz

**Comment by box293** [comment]
@prakashveni1 can you please create a forum thread here https://support.nagios.com/forum/viewforum.php?f=7 please, this doesn't seem the area to troubleshoot your problem.

**Comment by bardahlm** [comment]
Change introduced in #139 should be reverted.

**Comment by Rohlik** [comment]
Output `CRITICAL - Plugin timed out` is not informative. This ouput should be change to something like `PING CRITICAL - failed to ping host $IP`.

**Comment by edperry** [comment]
Yes we just realized what this was about, Plugin timeout it completely not helpful

**Comment by sawolf** [comment]
The fix for #139 has been reverted in `master`. If any of you still have this issue after re-compiling, please let me know and I'll re-open this issue.
